examples/loop_optimizations_service/service_py/BUILD

```diff
@@ -8,21 +8,19 @@ py_binary(
     name = "loops-opt-service-py",
     srcs = ["loops_opt_service.py"],
     data = [
+        "//compiler_gym/third_party/autophase:compute_autophase-prelinked",
+        "//compiler_gym/third_party/programl:compute_programl",
```

Comment:
Building on Linux is failing as I get this error (see CI log [here](https://github.com/facebookresearch/CompilerGym/runs/5417482434?check_suite_focus=true)):
```
-- Configuring done
CMake Error at build_tools/cmake/cg_macros.cmake:281 (add_dependencies):
  The dependency target
  "compiler_gym__third_party__autophase_compute_autophase" of target
  "examples__loop_optimizations_service__service_py__loops_opt_service" does
  not exist.
Call Stack (most recent call first):
  build_tools/cmake/cg_py_library.cmake:78 (cg_add_data_dependencies)
  examples/loop_optimizations_service/service_py/CMakeLists.txt:8 (cg_py_library)


CMake Error at build_tools/cmake/cg_macros.cmake:281 (add_dependencies):
  The dependency target
  "compiler_gym__third_party__programl_compute_programl" of target
  "examples__loop_optimizations_service__service_py__loops_opt_service" does
  not exist.
Call Stack (most recent call first):
  build_tools/cmake/cg_py_library.cmake:78 (cg_add_data_dependencies)
  examples/loop_optimizations_service/service_py/CMakeLists.txt:8 (cg_py_library)


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
Error: Process completed with exit code 1.
```

I tried many times to fix it, and with the issue that CMake configuration takes multiple hours (see #595 ), it was really difficult to solve it. So I merged for now and will fix it later.

Cc @ChrisCummins @sogartar